src/express/spike/controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from 'express';
+import axios from 'axios';
+import * as https from 'https';
+import config from '../../config';
+import { getTokenRequestHeaders } from '../../utils/headersHandler';
+
+class SpikeController {
+    static async redirectUser(_req: Request, res: Response) {
+        const redirectUrl = `${config.spike.redirectUrl}?\
```

Comment:
needs better indentation

Reply:
it would be much neater to define the query params as an object.

Reply:
the string is too longa and the lack of indentation is required because it will add spaces if I add indendation.

Reply:
I understand the reason you've written it so. Yet, it should not look like that.

Reply:
You can try putting all of those parameters in an object called `params` for instance. Then concatenating the output of the code below to the URL:
```javascript
Object.keys(params).map(key => key + '=' + params[key]).join('&');
```

Reply:
added